examples/nodejs/cache/doc-example-files/working-with-files.ts

```diff
@@ -0,0 +1,76 @@
+import * as fs from 'node:fs';
+import {CacheClient, CacheGetResponse, CacheSetResponse, Configurations, CredentialProvider} from '@gomomento/sdk';
+
```

Comment:
Are we going to commit this file? Otherwise main just bails without error.

Reply:
? sorry i'm not following your question

Reply:
oh you're asking about the json file. hrm let me look

Reply:
this example is really for the docs site, but i can add that file.

Reply:
Woops. Yes I meant the json file. As is we can syntax check but if we run the program, without the file, the run function will return fast.